### PR TITLE
Use logging framework for STDOUT calls

### PR DESCRIPTION
**Pull request by zundertj**
I hope you are oke with basically redoing https://github.com/JuliaCI/Coverage.jl/pull/156? Not sure Manifest changes should be in here, they were added when adding Logging.

Effectively also closes https://github.com/JuliaCI/Coverage.jl/issues/92.

### CODE REVIEW

**Comment by zundertj** [comment]
I have resolved the merge conflicts. I do see there is an error related to MbedTLS, which I believe has nothing to do with this PR. Could anyone review this PR, please?

File: Project.toml [hunk 1]
```diff
@@ -7,6 +7,7 @@ version = "0.8.1"
 HTTP = "cd3eb016-35fb-5094-929b-558a96fad6f3"
 JSON = "682c06a0-de6a-54ab-a142-c8b1cf79cde6"
 LibGit2 = "76f85450-5226-5b5a-8eaa-529ad045b433"
+Logging = "56ddb016-857b-54e1-b83d-db4d58db5568"
```
Comment (ararslan) [inline Project.toml]: The dependency on Logging (and by extension the `using Logging` occurrences) isn't actually necessary, as the basic logging macros—`@info`, `@warn`, and `@error`—are available in Base.

**Comment by ararslan** [comment]
> I do see there is an error related to MbedTLS, which I believe has nothing to do with this PR

Correct, that's https://github.com/JuliaWeb/MbedTLS.jl/issues/193.

File: Project.toml (hunk 1 above)
Comment (zundertj) [reply Project.toml]: Got it, removed the entry and the using statements.

**Pull request merged**